targets/Program.cs

```diff
@@ -23,11 +24,12 @@ class Program
         static void Main(string[] args)
         {
             
-            string RootDir = "";
+            string RootDir = "", gitHash = "";
             bool gitExists = true;
             try 
             {
                 RootDir = Read("git", "rev-parse --show-toplevel", noEcho: true).Trim(); 
+                gitHash = Read("git", "rev-parse --short HEAD", noEcho: true).Trim();               
```

Comment:
do we want the short or long hash?

Reply:
May as well be long hash since this is all tooling.